### PR TITLE
Feat/dot keypair

### PR DESCRIPTION
**Pull request by gibrangul95**
@fdmirza @argjv

### CODE REVIEW

**Review by jrafael-simtlix — APPROVED** [review]

File: modules/account-lib/package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "prepare": "npm run gen-protobuf && npm run gen-protobufts &&  npm run hedera-gen-protobuf && npm run hedera-gen-protobufts && tsc && cp -r ./resources ./dist",
     "unit-test": "nyc -- mocha --opts test/mocha.opts \"test/unit/**/*.ts\"",
     "test": "npm run unit-test",
-    "unprettied": "grep -R -L --include '*.ts' --include '*.js' --include '*.json' '@prettier' src test"
+    "unprettied": "grep -R -L --include '*.ts' --include '*.js' --include '*.json' '@prettier' src test",
+    "test-dot": "nyc -- mocha --opts test/mocha.opts \"test/unit/coin/dot/**/*.ts\""
```
Comment (argjv) [inline modules/account-lib/package.json:28]: This is non-standard, please remove and have it set up locally

File: modules/account-lib/package.json [hunk 2]
```diff
@@ -41,6 +42,8 @@
     "@ethereumjs/common": "^2.4.0",
     "@ethereumjs/tx": "^3.3.0",
     "@hashgraph/sdk": "^2.0.17",
+    "@polkadot/api": "^6.6.1",
```
Comment (argjv) [inline modules/account-lib/package.json:45]: Where are you using this dependency?

Comment (gibrangul95) [reply modules/account-lib/package.json:45]: gives us @polkadot/util-crypto @polkadot/types @polkadot/util which we use

File: modules/account-lib/src/coin/dot/keyPair.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  getAddress(format?: AddressFormat): string {
+    return this.createPolkadotPair().address;
+  }
+
+  /**
+   * Getting the KeyringPair for signing a dot transaction.
+   *
+   * @returns {KeyringPair} dot KeyringPair
+   */
+  getSigningKeyPair(): KeyringPair {
```
Comment (argjv) [inline modules/account-lib/src/coin/dot/keyPair.ts:44]: What's the difference between the signing key and `his.keyPair.prv;`?

Comment (gibrangul95) [reply modules/account-lib/src/coin/dot/keyPair.ts:44]: Returns an instance of keyring pair which has other methods and data based of off the private and public key

**Pull request closed without merging**